### ISSUE TITLE
Cannot unpin the window

### ISSUE THREAD

**Issue by NBOOK** [xupefei/QuickLook#118]
Windows 10 1709, QuickLook 0.3.2.1

**Comment by xupefei** [comment]
Yes, you are not able to unpin a window once it has been pinned.
It is design as such.


cheers,
Paddy

**Comment by xupefei** [comment]
Original
理由如下
只有一个窗口的时候，点击一次是固定，点第二次是取消固定
这时可以用其他内容覆盖窗口
当已经有了其他窗口的时候，点击相当于销毁该窗口，虽然和右上角“X”按钮功能一样，但是如果点一次就不可点“撤销”了，给我的感觉更差，甚至有点反人类，希望您弄两个更具有解释性的图标来表明功能的意思“贴图”“销毁”，以达到完美过度的目的
Version: 0.3.2.0 Country/Region: China OS: Windows 10 Device type: PC
 Update your response
